source/guides/dashboard/analytics.md

```diff
@@ -5,16 +5,10 @@ The Cypress Dashboard provides Analytics to offer insight into metrics like runs
 
 {% imgTag /img/dashboard/analytics/dashboard-analytics-overview.png "Dashboard Analytics Screenshot" %}
 
-{% note info %}
-Dashboard Analytics is currently in Beta.
-{% endnote %}
-
```

Comment:
The lines below this can go away too: it'll be enabled by default for all organizations.